### ISSUE TITLE
Uncaught ReferenceError after defer primefaces and CombinedResourceHandler 

### ISSUE THREAD

**Issue by jlerbsc** [omnifaces/omnifaces#310]
I have exactly the same issue using the exemple declared in https://github.com/omnifaces/omnifaces/issues/88 if we use CombinedResourceHandler with deferredScript (with primefaces 5.3.25 and omnifaces 1.11)
Uncaught ReferenceError: PrimeFaces is not defined
Uncaught ReferenceError: DeferredPrimeFaces is not defined

**Comment by jlerbsc** [comment]
As it is said in the previous issue (https://github.com/omnifaces/omnifaces/issues/88 ), if we remove target attribute for deferred omnifaces script, CombinedResourceHandler does not take into account and Uncaught ReferenceError no longer appear.

**Comment by BalusC** [comment]
Hmm, works just fine for me. What exactly is being combined in the combined script resource in the head? What's the combined resource URL? When I take over the code in #88, I get http://localhost:8088/playground/javax.faces.resource/eNrLz83LTEtMTi22yoex9LKKawqKMnNTIbyU1LTUoqLUFFRhq6zC0tSiSn0IBZQDADU3HRQ.js.xhtml?ln=omnifaces.combined&v=1419938518000 which contains omnifaces:omnifaces.js, primefaces.deferred.js (I didn't put it in a library) and primefaces:jquery/jquery.js.

**Comment by jlerbsc** [comment]
The combined resource URL is /adinfrance/javax.faces.resource/eNpdjEkKgDAQBF-kc_c3g05kYjazGAM-XiVoIKcuqKKdZ00CZwqT3BP5AnUGp9LKJowyXK41DTuhsNgUoU7nMkbyGv0GP3WFYEXJKYsLNHwbqw1_yXkw5RCfi8fcu8RJ6g.js?ln=omnifaces.combined&v=1455179488573 which contains
1/jQuery UI - v1.11.0 - 2014-07-22
Includes: core.js, widget.js, mouse.js, position.js, draggable.js, droppable.js, resizable.js, selectable.js, sortable.js, datepicker.js, slider.js, effect.js, effect-blind.js, effect-bounce.js, effect-clip.js, effect-drop.js, effect-explode.js, effect-fade.js, effect-fold.js, effect-highlight.js, effect-puff.js, effect-pulsate.js, effect-scale.js, effect-shake.js, effect-size.js, effect-slide.js, effect-transfer.js
2/Primefaces
3/OmniFaces.FixViewState

In the head section of the page

    <head>
    <script type="text/javascript" src="/adinfrance/javax.faces.resource/omnifaces.js?ln=omnifaces"></script>
    <script type="text/javascript" src="/adinfrance/javax.faces.resource/jquery/jquery.js?ln=primefaces&amp;v=5.3"></script>
    <link type="text/css" rel="stylesheet" href="/adinfrance/javax.faces.resource/eNorKMrMTU1LTE4ttiqAM_WSi4sBgtwJ8A.css?ln=omnifaces.combined&amp;v=1446199856000" />
    <script type="text/javascript">if(window.PrimeFaces){}</script>
    <meta name="viewport" content="initial-scale=1.0" />
    </head>


I have defined in web.xml context parameter COMBINED_RESOURCE_HANDLER_EXCLUDED_RESOURCES to exclude javascript

    <context-param>
		<param-name>org.omnifaces.COMBINED_RESOURCE_HANDLER_EXCLUDED_RESOURCES</param-name>
		<param-value>primefaces:jquery/jquery.js, default:js/primefaces-deferred-min.js, omnifaces:omnifaces.js</param-value>
	</context-param>

**Comment by BalusC** [comment]
I'm confused. A lot of those resources are not included in the example shown in #88. Perhaps you were not using exactly that example?

**Comment by BalusC** [comment]
> Yes i am using a custom jquery version

Why? PrimeFaces already auto-includes jQuery and jQuery UI. See also http://stackoverflow.com/questions/16166039/157882

Fix it and retry.

**Comment by jlerbsc** [comment]
Exactly the same result.
I do it because i want to test how to upgrade the jquery version if i need it.

**Comment by jlerbsc** [comment]
do you have an idea?

**Comment by BalusC** [comment]
I can't reproduce your problem. I suggest to re-apply the changes on a scratchpad project without touching any configuration defaults and/or duplicate jQuery libraries.

**Comment by jlerbsc** [comment]
Could you share your test project please. So i could look for differences between your  project and mine. Thanks.

**Comment by BalusC** [comment]
I didn't test your entire project, I just copied XHTML and bean from your project into a scratchpad project. Now I tested the provided WAR standalone and I could indeed reproduce your problem. But I was also seeing several new warnings and duplicate logs in Tomcat server log. 

Upon further inspection, it turns out that OmniFaces (and PrimeFaces) was loaded twice. Your concrete problem appears to be caused by the below unnecessary entry in `web.xml`,

```
	<listener>
		<listener-class>com.sun.faces.config.ConfigureListener</listener-class>
	</listener>
```

combined with a Tomcat-specific bug. 

After removing that entry from web.xml, everything worked fine for me. That listener is only necessary when deploying the webapp to a jurassic server. See also a.o. http://stackoverflow.com/q/20264083/157882
